### DISCUSSION TITLE
Hide 0mi / 0min Trips (summon events)

### DISCUSSION THREAD

**Discussion by cmchauvin** [teslamate-org/teslamate#1054] [Ideas]
**What would you like to be added**: When using summon, even for brief periods, a 0mi and 0 min trip is logged, skewing any drive averages. 

**Why is this needed**: Setting a minimum drive distance or travel time within dashboard views will prevent summon events from skewing summary drive statistics. 

**Additional context**:

**Comment by roadrash2108** [comment]
The "Efficiency" dashboard already has a filter similar to your requirement - if I understand you correct, you suggest such a filter should be added to other dashboards as well, right? If yes, to which dashboards in particular?

**Comment by cmchauvin** [comment]
For me, it appears that 0mi trips are included in “Trip” when selecting last 3 drives (I summon every day so this will always be the case for me). They also show in “Drives”..maybe that’s okay but would be good to filter them out since I really only to see legitimate drives. And since they show in Drives, I have to assume they contribute negatively to “Drive Stats” - specifically the median distance per day and the median distance of a drive.

**Comment by fetzu** [comment]
Similar to #618
